ui/apps/everest/src/components/cluster-form/advanced-configuration/advanced-configuration.tsx

```diff
@@ -68,10 +69,32 @@ export const AdvancedConfigurationForm = ({
       );
     }
   }, [clusterInfo]);
+
   const handleBlur = (value: string, fieldName: string, hasError: boolean) => {
     if (!hasError && !value.includes('/') && value !== '') {
       setValue(fieldName, `${value}/32`);
     }
+
+    // Check for duplicate entries
+    const existingEntries: Array<{ sourceRange?: string | undefined }> =
+      getValues(AdvancedConfigurationFields.sourceRanges) || [];
+    const isDuplicate = existingEntries.filter(
+      (item) => item.sourceRange === value
+    );
+    // Get unique entries to check if all are unique to clear the error
+    const uniqueEntries = Array.from(
```

Comment:
When using Zod, if we set an error whenever the user types a duplicate, it's guaranteed all entries will be unique, so we can skip all this section ahead